yarn.lock

```diff
@@ -1216,10 +1216,10 @@ Caret.js@INTELOGIE/Caret.js#0.3.1:
   dependencies:
     grunt "~0.4.1"
 
-"DOMPurify@npm:dompurify#2.3.6":
-  version "2.3.6"
-  resolved "https://registry.yarnpkg.com/dompurify/-/dompurify-2.3.6.tgz#2e019d7d7617aacac07cbbe3d88ae3ad354cf875"
-  integrity sha512-OFP2u/3T1R5CEgWCEONuJ1a5+MFKnOYpkywpUSxv/dj1LeBT1erK+JwM7zK0ROy2BRhqVCf0LRw/kHqKuMkVGg==
+"DOMPurify@npm:dompurify#2.5.0":
+  version "3.1.7"
```

Comment:
Hmm 🤔 This is weird, the version specified above is not same as the version it is fetching.

Reply:
yeah...which version does yarn install?

Reply:
I think it installed 3.1.7, I have gone through the changelog and I don't think anything has changed in the way `sanitize` should be used. And we only use `DOMPurify.sanitize`.
It has been on staging for a while and no issues were flagged. So I am going to bump it up to the 3.1.7 only.